phial/wrappers.py

```diff
@@ -143,3 +152,66 @@ def __init__(self,
 
     def __repr__(self) -> str:
         return "<Attachment in {0} >".format(self.channel)
+
+class MessageAttachment():
+    def __init__(self,
```

Comment:
Could you please add a docstring to this class?  This would mean that when docs are generated there is some information about what the class does and is used for. Feel free to borrow bits from the other docstring in this file.

Reply:
Don't know why this is outdated?